protokube/pkg/protokube/kube_boot.go

```diff
@@ -86,91 +88,92 @@ func (k *KubeBoot) RunSyncLoop() {
 }
 
 func (k *KubeBoot) syncOnce() error {
-	if k.Master {
-		volumes, err := k.volumeMounter.mountMasterVolumes()
-		if err != nil {
-			return err
-		}
+	// attempt to mount the volumes
+	volumes, err := k.volumeMounter.mountMasterVolumes()
```

Comment:
Good spot, I must have messed it up in the revert ... https://github.com/kubernetes/kops/compare/master...gambol99:etcd_tls?diff=split&name=etcd_tls#diff-eed2b06d1a16a4f2b372370d5795f8de